pkg/config/msp_test.go

```diff
@@ -23,7 +23,9 @@ import (
 )
 
 const (
-	// Arbitrary valid pem encoded x509 certificate
+	// Arbitrary valid pem encoded x509 certificate from crypto/x509 tests.
+	// The contents of the certifcate don't matter, we just need a valid certificate
+	// to pass marshaling/unmamarshaling.
```

Comment:
Really? I can give you an RSA cert and this will still work?

Reply:
I believe as long as it passes x509.ParseCertificate it will work. Not sure if an RSA cert would or not